src/main/java/org/cojen/util/BeanPropertyAccessor.java

```diff
@@ -122,21 +112,18 @@
     private static RuntimeClassFile generateClassFile(Class beanType, PropertySet set) {
         BeanProperty[][] props = getBeanProperties(beanType, set);
 
-        RuntimeClassFile cf = new RuntimeClassFile
-            (BeanPropertyAccessor.class.getName(),
-             BeanPropertyAccessor.class.getName(),
-             beanType.getClassLoader());
+        RuntimeClassFile cf = new RuntimeClassFile(BeanPropertyAccessor.class.getName(), BeanPropertyAccessor.class.getName(), beanType.getClassLoader());
         cf.markSynthetic();
         cf.setSourceFile(BeanPropertyAccessor.class.getName());
         cf.setTarget("1.5");
 
         MethodInfo ctor = cf.addConstructor(Modifiers.PUBLIC, null);
         ctor.markSynthetic();
-        CodeBuilder b = new CodeBuilder(ctor);
+        CodeBuilder codeBuilder = new CodeBuilder(ctor);
```

Comment:
This breaks a convention. CodeBuilder instances are named 'b' to make the code smaller and easier to read.